hive/runners/single_agent_loop.py

```diff
@@ -85,7 +85,9 @@ def __init__(
         )
         self._stack_size = stack_size
 
-    def run_one_step(self, environment, observation, episode_metrics, transition_info):
+    def run_one_step(
```

Comment:
This looks weird. Do you know why Black does that?

Reply:
I think it's because the line is longer now, and so it'd go above the 88 column threshold